backend/src/main/java/com/morak/back/core/support/LogFormatter.java

```diff
@@ -0,0 +1,38 @@
+package com.morak.back.core.support;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.Enumeration;
+import javax.servlet.http.HttpServletRequest;
+
+public class LogFormatter {
+
+    private static final String NEWLINE = System.getProperty("line.separator");
+
+    public static String toPrettyRequestString(HttpServletRequest httpRequest) throws IOException {
+        StringBuilder stringBuilder = new StringBuilder(NEWLINE);
+        appendMethodAndURI(httpRequest, stringBuilder);
+        appendHeaders(httpRequest, stringBuilder);
+        appendBody(httpRequest, stringBuilder);
```

Comment:
```suggestion
        stringbuilder.append(getMethodAndURI(httpRequest));
        stringbuilder.append(getHeaders(httpRequest));
        stringbuilder.append(getBody(httpRequest));
```

void 대신 return을 받아서 appned하는게 어떨까요 ?

Reply:
이러면 메서드 마다 StringBuilder를 생성해서 String을 만들어줘야하는데 괜찮은건가..?
```java
private static String getHeaders(HttpServletRequest httpRequest) {
    StringBuilder stringBuilder = new StringBuilder();
    Enumeration<String> headerNames = httpRequest.getHeaderNames();
    while (headerNames.hasMoreElements()) {
        String headerName = headerNames.nextElement();
        stringBuilder.append(headerName).append(": ").append(httpRequest.getHeader(headerName)).append(NEWLINE);
    }
    return stringBuilder.toString();
}
```